src/test/java/org/jilt/test/data/defaultvalue/DefaultValueWithOpt.java

```diff
@@ -0,0 +1,40 @@
+package org.jilt.test.data.defaultvalue;
+
+import org.jilt.Builder;
+import org.jilt.Opt;
+
+import java.util.Set;
+
+@Builder
+public class DefaultValueWithOpt {
+
+    @Opt
+    public int attr1 = 1;
```

Comment:
Let's not implement this functionality for `@Opt`. Let's make it work only for `@lombok.Builder.Default`.

I would remove this entire class.